lbmerchant93/petstrology
# Feature final tests/styling

What does this PR do?
Finish up testing
Finish up styling

What does it feature/fix?
Feature testing for header link conditional rendering
Testing for About page
Testing for Contact page
Fix background styling
Add Media Queries

Where should the reviewer start?
contact-spec.js line 1
about-spec.js line 1

How should this be tested?
All test should pass on Cypress

What are the next steps or any issues/concerns?
Deploy -> Fix routes in testing if deployed
Create Readme
